testing/setup/host-cluster.cloudformation.yaml

```diff
@@ -0,0 +1,61 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
I won't nitpick on these files much, since this will likely become a CDK component.

Reply:
Correct, these will probably change pretty quickly into CDK components.